lib/livebook/evaluator.ex

```diff
@@ -137,11 +137,17 @@ defmodule Livebook.Evaluator do
     context = Map.get_lazy(state.contexts, prev_ref, fn -> initial_context() end)
     file = Keyword.get(opts, :file, "nofile")
     context = put_in(context.env.file, file)
+    start_time = System.monotonic_time()
 
     {result_context, response} =
       case eval(code, context.binding, context.env) do
         {:ok, result, binding, env} ->
-          result_context = %{binding: binding, env: env}
+          result_context = %{
+            binding: binding,
+            env: env,
+            evaluation_time_ms: get_execution_time_delta!(start_time)
```

Comment:
We want to measure the time in case of an error as well, you can finish the measurement outside of this case block :)